easybuild/framework/easyconfig/types.py

```diff
@@ -116,3 +110,29 @@ def convert_value_type(val, typ):
         raise EasyBuildError("No conversion function available (yet) for target type %s", typ)
 
     return res
+
+def to_toolchain(tcspec):
+    """
+    Convert a toolchain string "foo, v0" to a dictionary {'name':'foo', 'version':'v0'}
+
+    @param tcspec: a toolchain in the form of a string or a list
+    """
+    # check if tc is a string or a list of two values; else, it can not be converted
+    if isinstance(tcspec, basestring):
+        tcspec = tcspec.split(',')
+
+    if isinstance(tcspec, list):
+        if len(tcspec) == 2:
+            return {'name':tcspec[0].strip(), 'version':tcspec[1].strip()}
```

Comment:
use `res = {...}` here, and `return res` at the end of the function (we try to stay away from 'internal' returns)

also, space after `:`